chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultWebChatStyles.ts

```diff
@@ -29,5 +29,30 @@ export const defaultWebChatStyles: StyleOptions = {
     showAvatarInGroup: true,
     suggestedActionsStackedHeight: 125,
     suggestedActionsStackedOverflow: "scroll" as any, // eslint-disable-line @typescript-eslint/no-explicit-any
-    typingAnimationDuration: 3500
+    typingAnimationDuration: 3500,
+    emojiSet: {
```

Comment:
It's unclear how this is going to fix anything, what is this referenced by?

Reply:
I have it Solution Proposed section , for reference 
https://github.com/microsoft/BotFramework-WebChat/blob/f03d1f05ea6bb87dda6bea6ab42f58b1b96d9dfa/packages/api/src/normalizeStyleOptions.ts#L36